pom.xml

```diff
@@ -1027,7 +1041,11 @@
                 <artifactId>jose4j</artifactId>
                 <version>${version.org.bitbucket.b_c.jose4j}</version>
             </dependency>
-
+            <dependency>
```

Comment:
Again, just a very minor thing, but would be good to add this dependency underneath the org.apache.httpcomponents:httpclient dependency.

Reply:
sure, will do